pycbc/frame/frame.py

```diff
@@ -906,16 +906,14 @@ def indices_of_flag(self, start_time, duration, times, padding=0):
         duration: int
             Number of seconds to check.
         padding: float
-            Number of seconds to add around flag inactive times to be considered
-        inactive as well.
+            Amount of time in seconds to flag around glitchy times
 
         Returns
         -------
-        indices: numpy.ndarray
-            Array of indices marking the location of triggers within valid
-        time.
+        flag_state: numpy.ndarray
+            Boolean array of whether flag was on at given times
         """
-        from pycbc.events.veto import indices_outside_times
+        from pycbc.events.veto import indices_within_times
         sr = self.idq.raw_buffer.sample_rate
         s = int((start_time - self.idq.raw_buffer.start_time - padding) * sr) - 1
```

Comment:
If you could add comments here as to what each block of a few lines is doing, that would help to understand. e.g. 
```
# Convert start time into sample number for the raw idq buffer
...
# Get the appropriate idq state for the start time and duration given
...
# Get the times when idq false alarm probability is below the threshold
...
# Apply padding around the glitch times
...
# Work out the indices of trigger times which fall into the padded flags
```